src/en/contact/contact.md

```diff
@@ -16,16 +16,16 @@ date: 'git Last Modified'
 
 
 <gcds-notice type="info" notice-title-tag="h2" notice-title="Support form on GitHub">
-  <gcds-text>With an <gcds-link external href="{{ links.githubGetStarted }}">account</gcds-link>, use our <gcds-link external href="{{ links.githubCompsIssues }}">GitHub support form</gcds-link> to report bugs and get technical help. You’ll have access to the team’s direct responses, progress made on your issue, and issues raised by others. </gcds-text>
+  <gcds-text>With an <gcds-link external href="{{ links.githubGetStarted }}">account</gcds-link>, use our <gcds-link external href="{{ links.githubCompsIssues }}">GitHub support form</gcds-link> to report bugs and get technical help. You'll have access to the team's direct responses, progress made on your issue, and issues raised by others. </gcds-text>
```

Comment:
I've been doing this too, as it's a common thing we developers often fix, but I've recently come to know that it is _intended_ and that `’` is the correct symbol to use for apostrophes and not `'`. Do you think we can keep it in our text from now on?

Reply:
Oh interesting, my IDE actually changed it but I can change it back.